lib/versioning/loose/utils.spec.ts

```diff
@@ -83,8 +88,9 @@ describe('versioning/loose/utils', () => {
       ${'isVersion'}       | ${true}
       ${'matches'}         | ${true}
       ${'sortVersions'}    | ${0}
-    `('$fn', ({ fn, expected }) => {
-      expect(api[fn]()).toBe(expected);
+    `('$fn', ({ fn, expected }: ApiTestData) => {
+      const apiFn: () => ApiTestExpected = api[fn].bind(api) as never;
```

Comment:
😕 looks pretty bad. We should split the test functions instead, otherwise we loose all type safety.